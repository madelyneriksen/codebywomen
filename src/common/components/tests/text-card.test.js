import React from 'react';
import renderer from 'react-test-renderer';
import TextCard from '../text-card';


describe("Text Card", () => {
  it("Matches Snapshot", () => {
    const data = {
      title: "The Zen of Programming",
      description: "The zen of programming",
    }
    const tree = renderer.create(<TextCard props={data} />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})
