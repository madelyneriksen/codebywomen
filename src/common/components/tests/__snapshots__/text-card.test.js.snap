// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Card Matches Snapshot 1`] = `
<div
  className="w-100 pa3"
>
  <h2
    className="accent-font ma0 navy"
  />
  <p
    className="sans-serif lh-copy f4 mid-gray"
  />
</div>
`;
